server/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -137,6 +137,20 @@
     </rule>
 
     <rule>
+        <name>Rails Config Repo API Index API</name>
```

Comment:
I think all url beginning with api are already forwarded to rails. Can you add config-repo to that rule itself?